### ISSUE TITLE
Move repository to a new organization

### ISSUE THREAD

**Issue by mrocklin** [mrocklin/tutorials#2]
I suggest that this might eventually move to the pydata organization.  

cc @aterrel @rgommers

**Comment by rgommers** [comment]
It would be quite useful to have a central site with high-quality tutorials that's easy to discover. Right now the best such thing is http://www.scipy-lectures.org/ (cc @GaelVaroquaux). Expanding that site with a section of links to Binder tutorials, videos, etc. would be the other option that comes to mind.

I'm fine with moving it under PyData in principle, but would prefer to do that once there's a bit more content and it's clear that this will be maintained.

Thinking even a bit bigger, I'd really like a better "gateway to the ecosystem" site. Right now http://scipy.org is limited, and it doesn't attract enough outside contributions/maintenance living under the `scipy` GitHub org. Then there's domain-specific things like http://pyviz.org/, which was rightfully criticized at SciPy'18 for not being inclusive while claiming all of "Python visualization" with its name.

Maybe we need a PyData ecosystem site, as a superset of the "SciPy ecosystem". Enough ideas; needs a few people with fresh energy to push them forward though.

**Comment by GaelVaroquaux** [comment]
> Right now the best such thing is http://www.scipy-lectures.org/ (cc
> @GaelVaroquaux). Expanding that site with a section of links to Binder
> tutorials, videos, etc. would be the other option that comes to mind.

The new version of sphinx-gallery (used partly to generate
scipy-lectures) can create a binder infrastructure. It's just a question
of doing the work so that scipy-lectures benefits from it.

That said, scipy-lectures aims to be teaching materials, and not overall
promotion.

But we would be happy to accept new chapters on packages that are not in
there you.

**Comment by mrocklin** [comment]
> I'm fine with moving it under PyData in principle, but would prefer to do that once there's a bit more content and it's clear that this will be maintained.

Fully agreed.  So far this hasn't lived longer than a single flight (so far this is an airport/airplane project :))  This also doesn't have to go under PyData, but if it does survive then it probably shouldn't live under my org.

> Right now the best such thing is http://www.scipy-lectures.org/ (cc @GaelVaroquaux). Expanding that site with a section of links to Binder tutorials, videos, etc. would be the other option that comes to mind.

> But we would be happy to accept new chapters on packages that are not in there you.

The approach and content of scipy-lectures is somewhat different so I have a slight preference to have this evolve independently.  One major difference is this project isn't trying to generate or curate content directly, we're only trying to organize and expose existing tutorial materials and hope to push video links upstream.

> Thinking even a bit bigger, I'd really like a better "gateway to the ecosystem" site. 

I'm entirely in favor of this and probably willing to put some time behind it (assuming we're thinking roughly the same way) in the moderate future.  

> Then there's domain-specific things like http://pyviz.org/, which was rightfully criticized at SciPy'18 for not being inclusive while claiming all of "Python visualization" with its name

No dissent from me.  If I recall correctly that argument *did* come up when they were planning that site.  I don't fully remember the reasoning around why they chose to stay with the name though.  Presumably this came up at SciPy 2018 though, so you may be more up to date than I.

**Comment by mrocklin** [comment]
Also I wanted to note that I'm misusing the pydata name currently.  I've been hosting this under http://pydata-tutorials.readthedocs.io for the last few hours just to put something up.  If possible I'd like to keep this up there for the short term while we're playing with the idea and seeing if we can get some invovlement.  I'd be happy to move it elsewhere though if we'd like to protect the pydata name a bit more.  Medium-term I'd like to see this at somewhere like https://tutorials.pydata.org

**Comment by rgommers** [comment]
> Also I wanted to note that I'm misusing the pydata name currently. I've been hosting this under http://pydata-tutorials.readthedocs.io for the last few hours just to put something up. If possible I'd like to keep this up there for the short term while we're playing with the idea and seeing if we can get some invovlement. I'd be happy to move it elsewhere though if we'd like to protect the pydata name a bit more.

Since this is in line with the educational & community building mission of PyData, I don't see a problem with that for now.

>  Medium-term I'd like to see this at somewhere like https://tutorials.pydata.org

Sounds sensible. It will require a bit of thought on whether we want to link it directly to/via the pydata.org site, which is now about the conference and community.



>>    Thinking even a bit bigger, I'd really like a better "gateway to the ecosystem" site.

> I'm entirely in favor of this and probably willing to put some time behind it (assuming we're thinking roughly the same way) in the moderate future. 

Awesome. I'm thinking about the one obvious place to start if you want to get into any aspect of PyData - whether it's as a beginning user, contributor, educator, etc. I haven't thought much about format/shape, but in terms of scope and being "the one place", something like https://julialang.org/ for Julia.

Ideally it would match the perspective from us as devs of core and core-surrounding packages with a community perspective (cc @dutc, @ianozsvald).

> ...and probably willing to put some time behind it 

Me too. I really should drop old things before starting new ones, but SciPy'18 reminded me that this is quite important.

**Comment by rgommers** [comment]
Just a brief update: we had a good brainstorm on this topic at BIDS last week. We've reserved https://scientific-python.org/ and are in the process of putting together a motivation/background/scope-of-new-site at https://github.com/scientific-python/scientific-python.org/issues/1. 

Due to bandwidth issues it'll be at least a week I think before we're ready to broaden that discussion.

Cc @dhavide - this is what we were talking about yesterday.

**Comment by mrocklin** [comment]
FWIW I recommend avoiding the term "scientific".  This stack is used in
many fields outside of pure science. This was, to my knowledge, part of the
reason for creating the term PyData.

**Comment by rgommers** [comment]
Would be nice to have an even broader term, but it's not obvious which one. PyData is taken, `python-data` and `numerical-python` don't really work, and "data science" is not a broader term. Concrete suggestions welcome.

**Comment by mrocklin** [comment]
I would consider placing this repository under tutorials.pydata.org

The term PyData is currently mostly used by the conference series, but
presumably that might change.

**Comment by rgommers** [comment]
Yes for these tutorials that's a good name. For https://github.com/scientific-python/scientific-python.org/issues/1 though the aim is not just tutorials, but an overall entry-point to the data/scientific-python world. That cannot have a PyData subdomain, and I doubt we can or want to repurpose what pydata.org is about (= conferences/meetups).

**Comment by mrocklin** [comment]
I guess I'd like to explore the possibility that we do repurpose the
current pydata.org page to make it about more than just conferences and
meetups.

From the perspective of someone who tries to direct funding from businesses
to open source software my opinion is that anything with the word "science"
in the name is counter-productive.  It also doesn't fully encompass the
current user-base of the software stack.  People use these tools for many
domains outside of traditional science.  As an example I'll bring up city
planning with GIS tools like geopandas.

**Comment by mrocklin** [comment]
To be clear, I'd also be open to names that aren't PyData and I'll try to
think of other names.  I do think that long term we might want to broaden
the use of that term.  That could be a separate discussion though.

**Comment by rgommers** [comment]
@mrocklin fair enough. I'll discuss the PyData name within NumFOCUS. Logistically a new generic name is significantly easier I think, but then we have to come up with one first.

**Comment by aterrel** [comment]
I'm +1 to PyData.
